app/src/main/java/com/google/android/stardroid/activities/util/FullscreenControlsManager.java

```diff
@@ -87,18 +89,21 @@ public void delayHideTheControls() {
   private final View.OnTouchListener mDelayHideTouchListener = new View.OnTouchListener() {
     @Override
     public boolean onTouch(View view, MotionEvent motionEvent) {
-      if (AUTO_HIDE) {
+      //Note this function should not be executed when hide_show_toolbar button is clicked
+      if (autoHideEnabled()) {
         delayedHide(AUTO_HIDE_DELAY_MILLIS);
       }
       return false;
     }
   };
 
   private void toggle() {
-    if (mVisible) {
-      hide();
-    } else {
-      show();
+    if (autoHideEnabled()) {
```

Comment:
Great. Can you remove the AUTO_HIDE constant too if it's no longer used?

Reply:
yeah sure

Reply:
Thanks! I don't mind doing that myself in a follow up, but it seemed to me it belongs in this change.  Appreciate the contribution.